src/Beman/Optional26/tests/detail/iterator.t.cpp

```diff
@@ -0,0 +1,178 @@
+/**
```

Comment:
This file needs license information per [FILE.LICENSE_ID].

Reply:
Please check - updated entire tree in 416ec70bd1e9da43702b29f4c01a231d22c2069d